### PR TITLE
Finetune ssd, yolo faster-rcnn.

### PR DESCRIPTION
**Pull request by sufeidechabei**

### CODE REVIEW

File: .idea/inspectionProfiles/Project_Default.xml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<component name="InspectionProjectProfileManager">
```
Comment (zhreshold) [inline .idea/inspectionProfiles/Project_Default.xml]: remove this file

**Comment by mli** [comment]
Job PR-280-2 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-280/2/index.html 
Code coverage of this PR:  vs. Master:

**Comment by mli** [comment]
Job PR-280-3 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-280/3/index.html 
Code coverage of this PR:  vs. Master:

**Comment by mli** [comment]
Job PR-280-4 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-280/4/index.html 
Code coverage of this PR:  vs. Master:

**Comment by mli** [comment]
Job PR-280-5 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-280/5/index.html 
Code coverage of this PR:  vs. Master:

**Comment by mli** [comment]
Job PR-280-6 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-280/6/index.html 
Code coverage of this PR:  vs. Master:

**Comment by mli** [comment]
Job PR-280-7 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-280/7/index.html 
Code coverage of this PR:  vs. Master:

**Comment by mli** [comment]
Job PR-280-8 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-280/8/index.html 
Code coverage of this PR:  vs. Master:

**Comment by mli** [comment]
Job PR-280-9 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-280/9/index.html 
Code coverage of this PR:  vs. Master:

**Comment by mli** [comment]
Job PR-280-10 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-280/10/index.html 
Code coverage of this PR:  vs. Master:

**Comment by mli** [comment]
Job PR-280-11 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-280/11/index.html 
Code coverage of this PR:  vs. Master:

**Comment by mli** [comment]
Job PR-280-13 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-280/13/index.html 
Code coverage of this PR:  vs. Master:

**Comment by mli** [comment]
Job PR-280-14 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-280/14/index.html 
Code coverage of this PR:  vs. Master:

**Comment by zhreshold** [comment]
closed with #319

**Pull request closed without merging**